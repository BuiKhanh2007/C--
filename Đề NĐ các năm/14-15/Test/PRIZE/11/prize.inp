7639235659516141799761189486149251213792634416254768384444615874756335528425216692541641826278247798575599445827335267863762976592297
1665791278537573321376864831462457124588259789926548731721574634337192923327979554337393645883341754515939641471262575288186433839232