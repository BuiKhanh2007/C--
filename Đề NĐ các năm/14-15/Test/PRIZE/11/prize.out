9305026938053715121138054317611708338380894206181317116166190509093528451753196246879035472161589553091539087298597843151949410431529
9